public/app/plugins/datasource/elasticsearch/datasource.ts

```diff
@@ -380,6 +380,9 @@ export class ElasticDatasource {
     }
 
     if (query.find === 'terms') {
+      if ('field' in query) {
```

Comment:
is this conditional required? terms always have a field property

Reply:
they do it the user enters it, if not it results in a field `field=undefined` which doesn't seem to cause any problems as the query is invalid.